gnmvidispine/vidispine_api.py

```diff
@@ -415,7 +417,7 @@ def request(self,path,method="GET",matrix=None,query=None,body=None, accept='app
         if raw_body.__len__() > 0:
             try:
                 if accept=='application/xml':
-                    return ET.fromstring(raw_body)
+                    return ET.fromstring(unicode(raw_body,errors='ignore'))
```

Comment:
can you change errors to 'xmlcharrefreplace'? double-check that is the right value on the python unicode documentation site